app/models/activity_insight_oa_file.rb

```diff
@@ -1,6 +1,8 @@
 # frozen_string_literal: true
 
 class ActivityInsightOAFile < ApplicationRecord
+  after_destroy :remove_file_download_location!
```

Comment:
I think it would be a good idea to test the removal of the file and the file_download_location metadata  within the `ActivityInsightOAFile` tests now that this is here.

Reply:
I added a test for the removal of the file directory, but I'm not sure how to test for the metadata since it's attached to the `ActivityInsightOAFile` & that's being destroyed. Please let me know if I'm missing something

Reply:
Ah whoops, that's right.  I don't think there would be a way to test to removal of the metadata.